drizzlepac/haputils/astrometric_utils.py

```diff
@@ -897,9 +895,18 @@ def find_fwhm(psf, default_fwhm, log_level=logutil.logging.INFO):
         log.warning("The PHOT_RESULTS table has no rows. Trying again.")
         return None
 
+    # Check the 'flags' column has at least one row with a flag value of zero
+    if (phot_results['flags'] == 0).sum() == 0:
+        log.warn("The PHOT_RESULTS table has no good rows. Trying again.")
+        return None
```

Comment:
Is this change due to the changed fitter or fixing another failure?
Otherwise looks good.

Reply:
The added check is **_not_** due to the changed fitter.  It is a check that should have been present all along.